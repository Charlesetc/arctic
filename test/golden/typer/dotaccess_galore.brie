define main [ [
 this.that.you
 are.too.cool
] ]
