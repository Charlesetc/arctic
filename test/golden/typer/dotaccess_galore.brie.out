root[(ident(define):: ident(main):: block[][(block[][(ident(this)::.that::.you::)::, (ident(are)::.too::.cool::)::]::Function)::Function]::Function)::]::